SCP -> BTP, Cloud Foundry - Cloud Foundry environment (#199)
SAP/cloud-sdk: docs/java/guides/tutorial-overview-sdk-java.mdx

Missions are the first point of contact if you want to familiarize yourself with the SAP Cloud SDK for Java and want to achieve certain goals.

> - [Create a Cloud Foundry environment App Using SAP Cloud SDK](https://developers.sap.com/mission.cloudsdk-cf-app.html)

Author: ```suggestion
- [Create an App in Cloud Foundry environment Using SAP Cloud SDK](https://developers.sap.com/mission.cloudsdk-cf-app.html)
```

  Create a Cloud Foundry environment app using SAP Cloud SDK, then add security and other advanced features like resilience and caching.

### Groups

Groups are a collection of topic-based tutorials.

> - [Create a Simple Cloud Foundry environment App Using SAP Cloud SDK](https://developers.sap.com/group.s4sdk-cloud-foundry.html)

Author: ```suggestion
- [Create a Simple App in Cloud Foundry environment Using SAP Cloud SDK](https://developers.sap.com/group.s4sdk-cloud-foundry.html)
```

   Set up your system to create an SAP Business Technology Platform application with the SAP Cloud SDK.

> 2. [Create a Sample Application on Cloud Foundry environment Using SAP Cloud SDK](https://developers.sap.com/tutorials/s4sdk-cloud-foundry-sample-application.html)

Reviewer: Here it also doesn't really make sense. It should be "on Cloud Foundry" or "in the Cloud Foundry environment".

Author: Here it makes sense. I'm not sure in this virtual sense if it's in or on. No difference for me either.

> 2. [Create a Sample Application on Cloud Foundry environment Using SAP Cloud SDK](https://developers.sap.com/tutorials/s4sdk-cloud-foundry-sample-application.html)

Author: ```suggestion
2. [Create a Sample Application in Cloud Foundry environment Using SAP Cloud SDK](https://developers.sap.com/tutorials/s4sdk-cloud-foundry-sample-application.html)
```

   Create and deep insert functionality for OData as supported by the SAP Cloud SDK.

> 9. [Create Cloud Foundry environment App with Blockchain Features of SAP Cloud SDK](https://developers.sap.com/tutorials/blockchain-sdk-sample.html)

Reviewer: I think here environment doesn't make sense - I saw some other places where this is the case. I think we would have to go through each occurence and check whether the sentence still makes sense...

> 9. [Create Cloud Foundry environment App with Blockchain Features of SAP Cloud SDK](https://developers.sap.com/tutorials/blockchain-sdk-sample.html)

Author: ```suggestion
9. [Create an App in Cloud Foundry environment with Blockchain Features of SAP Cloud SDK](https://developers.sap.com/tutorials/blockchain-sdk-sample.html)
```
